x/concentrated-liquidity/types/params.go

```diff
@@ -18,9 +20,10 @@ func ParamKeyTable() paramtypes.KeyTable {
 	return paramtypes.NewKeyTable().RegisterParamSet(&Params{})
 }
 
-func NewParams(authorizedTickSpacing []uint64) Params {
+func NewParams(authorizedTickSpacing []uint64, authorizedSwapFees []sdk.Dec) Params {
 	return Params{
 		AuthorizedTickSpacing: authorizedTickSpacing,
+		AuthorizedSwapFees:    authorizedSwapFees,
```

Comment:
Are we planning to have these be controlled via governance? Currently, we directly hard-code them. If we are going to be set on certain values pre-launch that are going to remain constant, I don't think there is a need for SDK params. We can keep them as Go-defined constants

Reply:
Or should I just hardcode them here like mint does and then its not needed in the upgrade handler at all

https://github.com/osmosis-labs/osmosis/blob/641a9a8ef0ea4320e21f513b788a3dd91f325cb2/x/mint/types/params.go#L53-L70

Reply:
This works!